Car_Localizer/Car_Localizer/Base_loss_model/loss_model_base.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class LossBase(nn.Module):
+    def __init__(self):
+        super(LossBase, self).__init__()
+
+        self.Alpha = nn.Parameter(torch.ones(1), requires_grad=True)
+        self.Beta = nn.Parameter(torch.zeros(1), requires_grad=True)
+        self.Sigma = nn.Parameter(torch.ones(1), requires_grad=True)
+
+    def my_bce_with_logits_loss(x, y):
+        loss = -1.0 * (y * F.logsigmoid(x) + (1 - y) * torch.log(1 - torch.sigmoid(x)))
```

Comment:
please refer to my comments according to explicit types e.g.,
`x:TensorType` and function return type
if you deal with floating points, please use 1.0 instead of 1, which will be treated implicitly as integer